data_structures/src/chain.rs

```diff
@@ -4037,4 +4092,103 @@ mod tests {
         assert_eq!(calculate_backup_witnesses(11, 3), 22);
         assert_eq!(calculate_backup_witnesses(11, 4), 44);
     }
+
+    #[test]
+    fn test_ordered_alts_no_tie() {
+        let mut trs = TotalReputationSet::new();
+        let mut alt_keys = AltKeys::default();
+
+        let p1 = PublicKey::from_bytes([1; 33]);
```

Comment:
This can be confusing because the keys are ordered by pkh, not by public key. So how about creating the pkh directly?

```suggestion
        let p1_pkh = PublicKeyHash::from_bytes(&[0x01 as u8; 20]).unwrap();
```

Reply:
very true